.github/workflows/codeql-disable-default.yml

```
@@ -0,0 +1,29 @@
+      actions: read
+      contents: read
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+
+    - name: Disable Default CodeQL Setup
+      uses: github/codeql-action/init@v3
+      with:
+        languages: javascript-typescript
+        disable-default-queries: true
```

actionlint (1.7.7)

input "disable-default-queries" is not defined in action "github/codeql-action/init@v3". available inputs are "build-mode", "config", "config-file", "db-location", "debug", "debug-artifact-name", "debug-database-name", "dependency-caching", "external-repository-token", "languages", "matrix", "packs", "queries", "ram", "registries", "setup-python-dependencies", "source-root", "threads", "token", "tools", "trap-caching"

(action)